zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -134,17 +134,48 @@ public static void main(String[] args)
     System.exit(0);
   }
 
-
+  private DistributedResourcePool getResourcePool()
+  /* InterpreterGroup group,
+        Properties prop,
+        RemoteInterpreterEventClient client) */
+      throws TException {
+    if (resourcePool != null)
+      return resourcePool;
+    try {
+      Properties prop = interpreterGroup.getProperty();
+      //Happens during tests.
+      if (prop == null)
+        prop = new Properties();
```

Comment:
I know we don't have style check for this official but could we put single line into bracket too? It would help readability and prevent possible merge mistakes.
(same for L143)
http://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/